go.sum

```diff
@@ -1,12 +1,15 @@
+github.com/bradleyfalzon/ghinstallation/v2 v2.0.4/go.mod h1:B40qPqJxWE0jDZgOR1JmaMy+4AY1eBP+IByOvqyAKp0=
```

Comment:
Looks like https://github.com/google/go-github/pull/2240 added ghinstallation as a dep on go-github

Reply:
Just having a quick look, so we expect issues with this?

Reply:
No issues, I'm just noting the implicit dependency getting picked up

Reply:
Thanks for pointing this out!

Even though there's a circular module dependency, this should be fine so long as the ghinstallation and core go-github packages don't mix - because this in a separate examples package this is fine.

It is something to keep an eye on because this could cause some weird interactions with https://go.dev/ref/mod#minimal-version-selection picking more recent versions than intended, but as long as we maintain compatibility it's fine.
We can probably make some changes to completely prevent this tough - we can either break the go-github examples into a new submodule (I'll open a PR for this), or worst case we can remove the dependency here if it does end up causing problems.